packages/frontend/web/server/render.ts

```diff
@@ -13,7 +13,11 @@ export const render = ({ body }: express.Request, res: express.Response) => {
                 site: 'frontend',
                 page: 'Article',
                 NAV: CAPI.nav,
-                config: CAPI.config,
+                config: Object.assign(
+                    {},
```

Comment:
Is this the right place to add this field to the config object?

Reply:
Seems reasonable to me. I think @MatthewJWalls started a spike to get config from parameter store, but I guess this config wouldn't be based off that anyway